### PR TITLE
QCed CanFam

### PR DESCRIPTION
**Pull request by David-Peede**
Added notes to help with clarifications, but besides that everything looks good!

### CODE REVIEW

**Comment by David-Peede** [comment]
Would someone mind walking me through what went wrong here? I thought I did everything the same as my previous PRs.... I also tried to google the exit code, but was a tad confused on what actually went wrong... Thanks in advance for any guidance!

**Comment by grahamgower** [comment]
Hey @David-Peede. There are two things going on here.

- There are some long lines at lines 114 and 115. We check for long lines (among other things) with `flake8 --max-line-length 89` during the continuous integration checks (https://stdpopsim.readthedocs.io/en/latest/development.html#unit-tests).
- The stdpopsim repository has many changes in it that have occurred since you last pulled or cloned from the repository. So your pull request needs to be *rebased* against the latest branch (https://stdpopsim.readthedocs.io/en/latest/development.html#rebasing).

**Comment by David-Peede** [comment]
Thanks again for all your help @grahamgower I hopefully got it right this time!

File: stdpopsim/catalog/CanFam/__init__.py [hunk 1]
```diff
@@ -106,8 +107,16 @@
     generation_time=3,
     generation_time_citations=[
         # Everyone uses 3 years because everyone else uses it.
-        # It's likely higher, at least in wolves:
-        # https://pubs.er.usgs.gov/publication/70187564
+        # It's likely higher, at least in wolves: https://
+        # digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1304&context=usgsnpwrc
```
Comment (grahamgower) [inline stdpopsim/catalog/CanFam/__init__.py:111]: Long URLs and line length limits don't interact well. It's a pity this doesn't have a DOI. How about using this shorter URL instead? https://science.sciencemag.org/content/352/6290/1228/tab-e-letters

Comment (David-Peede) [reply stdpopsim/catalog/CanFam/__init__.py:111]: Yes, that one should work as well! The more I look at it the original link should work as well! I was just a tad confused why it was on the USGS website and not Science's..

Comment (David-Peede) [reply stdpopsim/catalog/CanFam/__init__.py]: I updated it with the new link1

**Comment by grahamgower** [comment]
Thanks @David-Peede!

**Pull request merged**